src/commands/utility/vote-points.js

```diff
@@ -13,13 +13,27 @@ module.exports = {
         .setName('user')
         .setDescription(translateLanguage('votePoints.userDescription'))
         .setRequired(true)
+    )
+    .addStringOption((option) =>
+      option
+        .setName('point-type')
+        .setDescription('Select the type of points to award')
+        .setRequired(true)
+        .addChoices(
+          { name: 'Normal Points', value: 'normal' },
+          { name: 'Boosted Points', value: 'boosted' }
+        )
     ),
   async execute(interaction) {
     const user = interaction.options.getUser('user') || interaction.user;
+    const pointType = interaction.options.getString('point-type');
+
+    pointType === 'boosted' ? tagIds.boostedPointTagId : tagIds.addPointTagId;
```

Comment:
maybe this can be deleted?

Reply:
No, I use that in logic.

![image](https://github.com/user-attachments/assets/18affbe5-ea34-4c51-a698-b23e9a4f7cec)